src/models/Filter.js

```diff
@@ -110,15 +110,16 @@ export default class Filter {
       where.verifyStatus = this.verifyStatus;
     }
 
-    let orCondition = false;
     const { verifyStatus, ...restFilter } = where;
 
     if (verifyStatus) {
       if (verifyStatus.length === 1) {
```

Comment:
This clause can be removed too.
I don't like the reuse of `where` here – we'd be better off with something clearer. (Sorry, I know it's not your code originally. This would just be a good time to tidy it up.)

In fact, we don't need to do process `verifyStatus` at all, and the `planterIds` can be simplified to:
```
const planterIds = this.planterId.split(',').map((item) => ({
  planterId: item.trim());
});
```
Then we can just return this:
```
return { ...restFilter, or: verifyStatus, or: planterIds };
```
We will just need to check how LoopBack copes with empty `or` arrays.

Reply:
I got your point.
Let me work on it. I'll let you know once I commit the new changes.

Reply:
@nmcharlton ,
I have done the refactoring.